UKHomeOffice/cerberus-service
# COP-10990 - Add test for filtering targets by assignee

## Description
Add test for filtering targets by assignee
https://support.cop.homeoffice.gov.uk/browse/COP-10990
## To Test
npm run cypress:test:local -- --spec cypress/integration/airpax/issue-airpax-tasks.spec.js

Should verify filter targets by Assignee
Should verify filter target journeys by Assignee 


## Developer Checklist

\* Required
